Description: Tell libibverbs to look in /usr/lib/libibverbs for plugin library
Author: Roland Dreier <roland@digitalvampire.org>

Index: libnes-deb.git/nes.driver
===================================================================
--- libnes-deb.git.orig/nes.driver	2011-07-07 11:30:09.967590028 -0700
+++ libnes-deb.git/nes.driver	2011-07-07 11:30:18.877461266 -0700
@@ -1 +1 @@
-driver nes
+driver /usr/lib/libibverbs/libnes
